src/settings/OrderTemplates/OrderTemplateView/OrderTemplateViewContainer.js

```diff
@@ -65,9 +62,8 @@ class OrderTemplateViewContainer extends Component {
   };
 
   render() {
-    const { close, match: { params: id }, resources, rootPath } = this.props;
-    const orderTemplatesList = getOrderTemplatesList(get(resources, ['orderTemplates', 'records'], []));
-    const orderTemplate = find(orderTemplatesList, id, {});
+    const { close, resources, rootPath } = this.props;
+    const orderTemplate = get(getOrderTemplatesList(get(resources, 'orderTemplate.records', [])), 0, {});
```

Comment:
I'm not sure that this should be using `getOrderTemplatesList`. I expect `orderTemplate` resource gets template id from browser url and pass it to backend for specific template, not the list.

Reply:
`orderTemplate` resource returns an object, where you can find an array of records (in this case an array with one record). But to use it,  I need to convert the value from JSON.